_examples/wrap_autoscroll.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2014 The gocui Authors. All rights reserved.
```

Comment:
Do we need this extra example?  
This seems more like a test for re-creating locally a problem.

Reply:
Yeah, you are right.... It kinda simulates the idea of autoscroll and wrap together.     

To be honest, you can recreate that by using `active.go` or `custom_frames.go` and makes your terminal small enough to force it to wrap the text in second view.    
So I can remove that exmple.